### PR TITLE
stylesheet

### PR DESCRIPTION
**Pull request by vikram-redhat**

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@adellape

File: _stylesheets/docs.css [hunk 1]
```diff
@@ -618,7 +619,7 @@ span.footnote a:active, span.footnoteref a:active { text-decoration: underline;
 div.unbreakable { page-break-inside: avoid; }
 code { color: #404040; background-color: #e7e7e7; font-weight: bold; font-family: "Roboto Mono", monospace;}
 h5 { color: #404040; }
-strong { color: #404040; font-weight: bold; }
+strong { color: #404040; font-weight: bold; font-family: "Roboto Mono", monospace; }
```
Comment (adellape) [inline _stylesheets/docs.css]: @vikram-redhat 	Can we revert this change?
